static/i18n/en/app.json

```diff
@@ -878,6 +879,10 @@
       "title": "That's the wrong app",
       "description": "Open the ‘{{managerAppName}}’ app on your device"
     },
+    "NoAccessToCamera": {
+      "title": "",
+      "description": "Please ensure your system has a camera, that it's not in use by another application and that Ledger Live has permission to use it."
```

Comment:
@dasilvarosa please confirm. thanks!

Reply:
@juan-cortes  I feel. we need to use title. not description. We have a guarantee in errors that a title is always set, only description is optional.

Reply:
The wording comes directly from @dasilvarosa , so very little trust..